### ISSUE TITLE
Kodi vaughnlive plugin

### ISSUE THREAD

**Issue by magog45** [HIGHWAY99/plugin.stream.vaughnlive.tv#3]
It seems that the plugin has quit working, I do see a post in another place about it but to date my post in the Kodi forums has no response. While it starts up fine and loads a channel list, nothing streams and sometimes Openelec freezes and needs a reboot.

**Comment by intrcomp** [comment]
Yes I received a message that vaughnlive addon is been blocked by the site because they stated that is affecting their revenue and their users. this is what I saw

 I appreciate the enthusiasm on using Vaughn Live, however this addon is costing us thousands of dollars per month. We cannot and will not support bypassing watching Vaughn Live with this addon as it greatly hurts us and our legitimate viewers. Unfortunately we have no choice but to continue breaking every work around that is made.
